pkg/kubelet/volume/gce_pd/gce_pd.go

```diff
@@ -196,7 +196,7 @@ func (pd *gcePersistentDisk) SetUp() error {
 	}
 
 	// Perform a bind mount to the full path to allow duplicate mounts of the same PD.
-	globalPDPath := makeGlobalPDName(pd.plugin.host, pd.pdName, pd.readOnly)
+	globalPDPath := makeGlobalPDName(pd.plugin.host, pd.pdName)
```

Comment:
Done.

Does DetachDisk need a similar treatment? It wasn't as obvious a win for the PD case, but I don't know what iscsi/others are going to look like.